index.js

```diff
@@ -82,6 +83,7 @@ function Bankai (entry, opts) {
     if (self.optimize) {
       b.transform(unassertify, { global: true })
       b.transform(yoyoify, { global: true })
+      b.transform(envify, { global: true })
```

Comment:
Yes to the first, no to the second:
```js
b.transform(envify({
  NODE_ENV: process.env.NODE_ENV
}))
```
Although I thought `envify` would take care of that out of the box, if you're not using `envify/custom`